### ISSUE TITLE
Tighter CCF UI Integration

### ISSUE THREAD

**Issue by bherr2** [hubmapconsortium/portal-ui#383]
Create a tighter CCF UI integration so that it can have a portal url (http//portal.hubmapconsortium.org/ccf-ui) and have key settings sent to it (nexus token and appropriate api endpoints). 

The CCF UI can now be configured with global settings. The below html template can be used for /ccf-ui with no other files needed to be hosted on the portal. Simply replace the values in the dbOptions dictionary. If you decide to host the ccf-ui files, you will only need to change the base href.

```html
<!doctype html>
<html lang="en">

<head>
  <script async src="https://www.googletagmanager.com/gtag/js?id=UA-136932895-2"></script>
  <script>
    window.dataLayer = window.dataLayer || [];

    function gtag() {
      dataLayer.push(arguments);
    }
    gtag('js', new Date());

    gtag('config', 'UA-136932895-2');

  </script>
  <meta charset="utf-8">
  <title>HuBMAP CCF Exploration User Interface (CCF-EUI)</title>
  <base href="https://cdn.jsdelivr.net/gh/hubmapconsortium/ccf-ui@portal-version/">
  <script>
    window.dbOptions = {
      hubmapDataService: 'search-api',
      hubmapDataUrl: 'https://search-api.test.hubmapconsortium.org/search',
      hubmapAssetsUrl: 'https://assets.test.hubmapconsortium.org',
      hubmapToken: 'xyz'
    };
  </script>
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <link rel="icon" type="image/x-icon" href="favicon.ico">
  <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500&display=swap" rel="stylesheet">
  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
  <link rel="stylesheet" href="styles.css">
</head>

<body class="mat-typography">
  <ccf-root></ccf-root>
  <script src="runtime-es2015.js" type="module"></script>
  <script src="runtime-es5.js" nomodule defer></script>
  <script src="polyfills-es5.js" nomodule defer></script>
  <script src="polyfills-es2015.js" type="module"></script>
  <script src="main-es2015.js" type="module"></script>
  <script src="main-es5.js" nomodule defer></script>
</body>

</html>
```

**Comment by mccalluc** [comment]
I'll get started on this, but some questions:
- It's been a while, but the google analytics won't work until we register our site with them, right?
- How are you using `hubmapDataService: 'search-api'`? It seems to be implicit in the other config.

**Comment by bherr2** [comment]
GA should just work as is. That's for tracking ccf-ui related activity. hubmapDataService can also be 'static' if there is a static dump of the data to use (which is what we're using in testing right now). It could theoretically be inferred based on the arguments given.

**Comment by bherr2** [comment]
Ok, I talked with Katy and the preference is to have the link text be "CCF" and the link url be /ccf-eui. Other than that, I think we're good to go!

**Comment by mccalluc** [comment]
Fixed in #385. Closing.
